src/components/Button/Button.Styles.ts

```diff
@@ -12,10 +13,13 @@ export const ButtonLayout = styled.button<{
   height: ${(props) => props.height};
   background-color: ${(props) => props.color};
```

Comment:
```suggestion
  background-color: ${({color}) => color};
```

[P3] 여기선 구조분해가 좀 더 가독성에 좋을것 같다는 의견 드려봅니다

Reply:
[P3]적용하신다면 전체다 해보시는것도 좋을 것 같숩다